### ISSUE TITLE
[Bug] Various alien race pawns cannot change ammo type or fire modes

### ISSUE THREAD

**Issue by sumghai** [AthenaSulisMinerva/CombatExtendedFastTrack#19]
Steam user **Spriggs** reported that none of the third-party alien races powered by the Humanoid Alien Races 2.0 framework can change ammo type or fire modes, which I have also confirmed:

* Xenohumans - Gen-spliced Xenos on the Rim
* Lizardmen
* Yabils: Space Gremlins

This seems to be a fairly serious bug, so I'm gonna have a crack at it tonight.

### Master List of Alien CE Patches that need fixing
- [x] Equium
- [x] Ferrex / Ferret
- [x] Forsakens
- [x] Kurin
- [x] Leeani
- [x] Lizardmen
- [x] Logann
- ~~Ni'Hal~~ _(current maintainer Zerithos XIII stated on mod page he will work on CE patches in his own time)_
- [x] Orassans
- [x] Racc
- [x] Sergals, Gnolls and Lions
- [x] Vulpine
- [x] Warhammer: Gor
- [x] Xenn
- [x] Xenohumans
- [x] Yabils

**Comment by sumghai** [comment]
After some experimenting...
* The Lizardmen and Yabils mods were authored by @Owlchemist, who directly bundled CE support into those particular race mods
* Since Owlchemist has made it clear he doesn't want to explicitly specify the name "Combat Extended: FastTrack Edition" in his patches, I ended up copying them into CE:FT
* I then found that he wrote up the CE Race compatibility definitions somewhat differently to how @Superlagg did the original ones currently included in CE:FT
* Finally, as a test, I rewrote the Orassan CE Race compatibility definition using @Owlchemist's format, and the Orassans are now able to freely change ammo type and fire modes

Basically, I'll just have to modify all the alien race patches we have so far to the new, valid format.

**Comment by AthenaSulisMinerva** [comment]
Shit, that sounds serious.

Thanks for taking care of this.

**Comment by sumghai** [comment]
It seems that the Equium mod was originally defined using an older method of HAR 2.0, hence my typical race compatibility definition rewrites can't be used. Instead, I've kept the original patch, but fixed Equium pawns not spawning with ammo (as well as consolidating many of the patches.

> Shit, that sounds serious.
>
> Thanks for taking care of this.

Yeah, I had to admit I was pretty caught off-guard that the alien race patches included in the original CE project had been broken for a while. Looks like there were some changes to either CE or HAR, and the patches were never updated after the fact.

In any case, the Equiums stuff is mostly working again, so I'm gonna keep going with these fixes.
